pages/establishment/update/index.js

```diff
@@ -17,6 +42,10 @@ module.exports = settings => {
     req.breadcrumb('establishment.update');
     req.model = groupFlags(req.establishment);
 
+    req.model.authorisationTypes = uniq(
+      req.model.authorisations.reduce((acc, authorisation) => acc.concat(authorisation.type), [])
```

Comment:
.map

Reply:
`req.model.authorisations.map(authorisation => authorisation.type)`

Reply:
yeah not sure what I was doing here!